### ISSUE TITLE
[Feature request] Input a value from stdin

### ISSUE THREAD

**Issue by tranleduy2000** [axkr/symja_android_library#148]

**Comment by axkr** [comment]
What do you mean exactly?
Do you have an example?

**Comment by tranleduy2000** [comment]
In console mode, when execute an expression like `a=input("prompt")`, `a` symbol will be set value read from keyboard.

**Comment by axkr** [comment]
`Input()` and `InputString()` should work, if `Config.FILESYSTEM_ENABLED = true` is enabled.

- `Input()` directly evaluates the input string
- `InputString()` only return the input string

**Comment by axkr** [comment]
Example:

```
DialogInput(
    DialogNotebook(
      Row({
          TextCell("Click the DoIt button:"),
          Button("DoIt", DialogReturn("Hello dialog world"))
      })
    )
  )
```

**Comment by tranleduy2000** [comment]
There are some problems when you implement this feature by using `java.awt.*` packages since Android doesn't has these classes. I will remove these features in Android library.

**Comment by tranleduy2000** [comment]
Android UI classes must be run on UI thread, but to make UI run smoothly, we must perform any calculation in worker thread.

**Comment by tranleduy2000** [comment]
```Java
interface DialogDelegate {
    /** @param locker an object to be call Object#wait() to wait for returning result */
    String showInputDialog(String title, String hint, String defaultValue, Object locker);
}
```

```Java
class Config {
    public static WeakReference<DialogDelegate> dialogDeletegteRef = null;
}
```

**Comment by axkr** [comment]
The current implementation will be released in 2.0.0. Please open new issue if necessary.
